src/frontend/komponenter/Behandling/Inngangsvilkår/Sivilstand/helper.ts

```diff
@@ -0,0 +1,26 @@
+import { SivilstandType } from '../../../../typer/personopplysninger';
+
+export const hentSivilstatus = (sivilstand: SivilstandType): string => {
```

Comment:
hentSivilstatus låter som att ska hämta noe på backend.. inte en mappning mellan enum och string

Reply:
@miichu Burde vi kanskje bruke Record? Å legge den ett annet sted som går å bruke for personopplysninger også
`Record<SivilstandType, string> = {`

Reply:
Evt `sivilstatusTilTekst` eller noe?